src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,36 @@
+const FETCHED_ROCKETS = 'app/rockets/FETCHED_ROCKETS';
```

Comment:
please use ROCKETS_FETCHED instead of FETCHED_ROCKETS to be according to convention